src/Enum/GlossarySubmissionEntryFormatEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scn\DeeplApiConnector\Enum;
+
+final class GlossarySubmissionEntryFormatEnum
+{
+    public const FORMAT_TSV = 'tsv';
```

Comment:
As we already have a dependency on >PHP8.1 here, we should use an enum here enum GlossarySubmis..: string {
case xxx
}

Reply:
Yes, I think it's in another PR. If you want I can help with this, but you need to update the php versions in composer before :)